src/classes/VOL_SharedCode.cls

```diff
@@ -948,6 +1005,8 @@ global with sharing class VOL_SharedCode {
         str = Label.labelFindVolunteersCriteria;
         str = Label.labelFindVolunteersHelpAssign;
         str = Label.labelVolunteersWizardNewCampaignTitle;
+        str = Label.labelContactLookupSuccess;
```

Comment:
I asked about this in the previous Pull Request but was it intentional not to have that merged into master and should this point to the same branch the work item for ambiguity pointed to?